R-package/README.md

```diff
@@ -20,14 +20,56 @@ Note: 32-bit (i386) R/Rtools is currently not supported.
 
 #### Windows Preparation
 
-Installing [Rtools](https://cran.r-project.org/bin/windows/Rtools/) is mandatory, and only support the 64-bit version. It requires to add to PATH the Rtools MinGW64 folder, if it was not done automatically during installation.
+Installing a 64-bit version of [Rtools](https://cran.r-project.org/bin/windows/Rtools/) is mandatory.
 
-The default compiler is Visual Studio (or [VS Build Tools](https://visualstudio.microsoft.com/downloads/)) in Windows, with an automatic fallback to Rtools or any [MinGW64](https://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/) (x86_64-posix-seh) available (this means if you have only Rtools and CMake, it will compile fine).
+After installing `Rtools` and `CMake`, be sure the following paths are added to the environment variable `PATH`. These may have been automatically added when installing other software.
 
-To force the usage of Rtools / MinGW, you can set `use_mingw` to `TRUE` in `R-package/src/install.libs.R`.
+* `Rtools`
+    - If you have `Rtools` 3.x, example:
+        - `C:\Rtools\mingw_64\bin`
+    - If you have `Rtools` 4.0, example:
+        - `C:\rtools40\mingw64\bin`
+        - `C:\rtools40\usr\bin`
+* `CMake`
+    - example: `C:\Program Files\CMake\bin`
+* `R`
+    - example: `C:\Program Files\R\R-3.6.1\bin`
+
+As of `Rtools` 4.0, some common paths changed and software was removed from `Rtools`. If you are using `R` 4.0 or later and the corresponding `Rtools`, you need to add one additional path to `PATH`.
+
+* `Rtools` usr bin:
+    - example: `C:\Rtools\usr\bin`
```

Comment:
Why not just add this path to the case ```If you have `Rtools` 4.0, example:```?

Reply:
huh I thought I did that already! sorry, will do

Reply:
oh! now I remember. That section you're referring to is documenting how the R package will fall back to msys2 if you don't have Visual Studio.

Adding this `/usr/bin` path is important specifically for the case of Visual Studio (the default), because that path is where `objdump.exe` is.

Reply:
Sorry, I didn't get it. I'm referring the following section:

![image](https://user-images.githubusercontent.com/25141164/84429561-08d87580-ac31-11ea-87e0-0ae81b5f788f.png)


Fallbacks are documented much later in the README.

Reply:
ooooooooh I misunderstood! Yes you're right, ok those two are definitely duplicates

Reply:
almost missed this one, sorry!  Just fixed this in https://github.com/microsoft/LightGBM/pull/3065/commits/d0e995f443531669c7e5faa9db8eec94b5ff0445.

I also added one more fix in that commit noticed during testing...if using `system2()` instead of `{processx}`, you have to `shQuote(args)` in because on of the args is a file path to `R.dll`, which might have spaces. This isn't necessary with `{processx}` because it does that quoting for you on Windows.